uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-record-actions.html

```diff
@@ -1,14 +1,26 @@
-<div class="btn-group" uib-dropdown>
-  <button type="button" class="btn {{vm.btnClass}} dropdown-toggle" uib-dropdown-toggle>
-    <span class="visible-sm visible-md visible-lg"><i class="fa fa-plus-circle" aria-hidden="true"></i> Record New Absence</span>
-    <span class="hidden-sm hidden-md hidden-lg"><i class="fa fa-plus" aria-hidden="true"></i> New</span>
+<div class="btn-toolbar">
```

Comment:
A toolbar is not correct here, from the [documentation](http://getbootstrap.com/docs/3.3/components/#btn-groups-toolbar)
> Combine sets of `<div class="btn-group">` into a `<div class="btn-toolbar">` for more complex components.
here you don't have a set of button groups, just one

Reply:
Done.